File: sol/2706.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        minEle, secMinEle = 101, 101
        for n in prices:
            if n < minEle:
                minEle, secMinEle = n, minEle
            elif n < secMinEle:
                secMinEle = n
        tot = minEle + secMinEle
        return money - tot if money >= tot else money

File: sol/2706/2706.cpp
struct Solution {
    int buyChoco(const vector<int>& prices, int money) {
        int minEle = 101, secMinEle = 101;
        for (const int &n : prices){
            if (n < minEle){
                secMinEle = minEle;
                minEle = n;
            } else if (n < secMinEle){
                secMinEle = n;
            }
        }
        minEle += secMinEle;
        return (minEle <= money) ? (money - minEle) : money;
    }
};
